JS-Algorithms/challenges.js

```diff
@@ -95,7 +113,24 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
```

Comment:
Just a comment, you could avoid the function in this exercise.
`const digitSum = (n) => {
  if (!(n > 1)) return;

  let result = 1;
  let total = 0;

  for (let i = n; i > 1; i--) {
    result *= i;
  }

  const digits = String(BigInt(result));

  for (const digit of digits) {
    total += Number(digit);
  }

  return total;
};`